d3d/HLSL_ShaderModel6_5.md

```diff
@@ -257,7 +257,7 @@ sum = WaveMultiPrefixSum(value, mask);
 // Is equivalent to writing a loop like this:
 while (true) {
   if (WaveReadLaneFirst(expr) == expr) {
-    sum = WaveActiveSum(value));
+    sum = WavePrefixSum(value));

```

Comment:
Just realized that the paragraph preceding this still says `WaveActiveSum` and needs updating to match the change in the example as well.